include/wx/defs.h

```diff
@@ -259,6 +259,19 @@ typedef short int WXTYPE;
     #define wxFALLTHROUGH ((void)0)
 #endif
 
+/* wxNODISCARD is used to notate that the function return value should not be ignored */
```

Comment:
I'm sorry, but is "notate" the right word to use here? I'd say "indicate". I'd also use "must" instead of "should".

Reply:
I just copied the comment from `wxFALLTHROUGH`, probably wrongly.

I will change the wording to what you suggested, once we agreed on other changes.